Display custom status on wishlist/checkout product configure pages (FR-397)

diff --git a/Plugin/Product/View/Type/Configurable.php b/Plugin/Product/View/Type/Configurable.php
index 53418f2..7cccde8 100644
--- a/Plugin/Product/View/Type/Configurable.php
+++ b/Plugin/Product/View/Type/Configurable.php
@@ -308,7 +308,15 @@ class Configurable

     protected function isProductPage(): bool
     {
-        return $this->request->getFullActionName() === 'catalog_product_view';
+        // BEGIN PATCH
+        //return $this->request->getFullActionName() === 'catalog_product_view';
+        $fullActionNamesArray = [
+                'catalog_product_view',
+                'wishlist_index_configure',
+                'checkout_cart_configure'
+            ];
+        return in_array($this->request->getFullActionName(), $fullActionNamesArray);
+        // END PATCH
     }

     protected function shouldLoadStock(): bool
--
2.39.5 (Apple Git-154)
